recipes/orange3/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "3.3.6" %}
+
+package:
+  name: orange3
+  version: {{ version }}
+
+source:
+  fn: Orange3-{{ version }}.tar.gz
+  url: https://github.com/biolab/orange3/archive/{{ version }}.tar.gz
+  md5: 927f3702ea5c9f1c8d1b59478476b4d5
+
+build:
+  number: 0
+  script:
+    # FIXME: This is a hack to make sure the environment is activated.
+    # The reason this is required is due to the conda-build issue
+    # mentioned below.
+    #
+    # https://github.com/conda/conda-build/issues/910
+    #
+    - source activate "${CONDA_DEFAULT_ENV}"  # [unix]
```

Comment:
There is nothing actionable for you, @astaric, AFAICT.

Yeah, see PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/14 ) for unpinning on OS X.

We need the same for Windows too as we are pinned on Python 3.4 32-bit still.